lib/editing/editor_ace.dart

```diff
@@ -115,6 +115,12 @@ class _AceEditor extends Editor {
     return new _AceDocument._(this, session);
   }
 
+  //TODO: Implement execCommand for ace.
+  void execCommand(String name) => null;
+
+  //TODO: Implement completionActive for ace.
+  bool get completionActive => null;
```

Comment:
We should probably return `false` here. Many clients will the return value in an if statement without checking for null; they'll get failures when run in checked mode. I'll patch this after merging.